include/mbgl/ios/MGLMapView.h

```diff
@@ -32,6 +32,7 @@ IB_DESIGNABLE
  *   @param styleURL The map style URL to use. Can be either an HTTP/HTTPS URL or a Mapbox map ID style URL (`mapbox://<user.style>`).
  *   @return An initialized map view, or `nil` if the map view was unable to be initialized. */
 - (instancetype)initWithFrame:(CGRect)frame accessToken:(NSString *)accessToken styleURL:(NSURL *)styleURL;
+- (instancetype)initWithFrame:(CGRect)frame accessToken:(NSString *)accessToken styleURL:(NSURL *)styleURL cache:(BOOL)enabled;
```

Comment:
Also, is there an end user common need for this initializer? They should be pared down to just the minimums to avoid confusion on different ways to create the map. Don't I want a cache? That sounds beneficial? Why wouldn't I cache? Does that mean file, memory, or both? These will show up in support.